src/components/AddBook.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Addbook = () => (
+  <div>
+    <h3>ADD NEW BOOK</h3>
+    <form>
+      <input type="text" placeholder="Book Title" />
+      <select id="category" name="category">
+        <option value="Category" selected disabled>Category</option>
+      </select>
+      <button type="button">Add Book</button>
+    </form>
```

Comment:
✅ Renders a form with inputs for the title and the author of the book 👍🏻 
✅ Has a button to submit 👍🏻